src/lib/utils.h

```diff
@@ -180,4 +180,14 @@ STORE64BE(uint8_t x[8], uint64_t y)
 #define RNP_USE_STD_REGEX 1
 #endif
 
+inline char *
+GETENV_LOGNAME(void)
+{
+	char *name = getenv("LOGNAME");
+    if (!name) {
+		name = getenv("USER");
+	}
```

Comment:
There is some misalign here, or mixed space/tabs. Could you please apply clang-format so it is correctly spaced?